_includes/payment-order-other-features.md

```diff
@@ -1429,6 +1429,70 @@ with the request parameter `instrument`.
 
 {% include settlement-reconciliation.md api_resource="paymentorders" %}
 
+## 3-D Secure 2
+
+When dealing with card payments, 3-D Secure authentication of the
+cardholder is an essential topic. 3-D Secure 2 is an improved version of the
+old protocol, now allowing frictionless payments where transactions can be
+completed without input from the cardholder. Therefore, there are certain fields
+that should be included when implementing 3-D Secure 2. These are listed below
+in the abbreviated `Purchase` request. 
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/paymentorders HTTP/1.1
+Host: {{ page.api_host }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "paymentorder": {
+        "operation": "Purchase",
+        "currency": "SEK",
+        "amount": 1500,
+        "vatAmount": 375,
+        "description": "Test Purchase",
+        "generatePaymentToken": true,
+        "userAgent": "Mozilla/5.0...",
+        "urls": {
+            "hostUrls": [ "https://example.com", "https://example.net" ]
+        },
+        "payer": {
```

Comment:
Thinking long about this, I don't think it's the `payer` object we should have here at all. In [the documentation](https://developer.swedbankpay.com/checkout/other-features#creating-a-payment-order) you can see that we have `riskIndicator` for `paymentorder` just like `payment`.

We can therefore just create an include from the information you now have in `payments/card/other-features.md`, which takes `api_resource` as an argument to change `paymentorder` with `payment` in the code example depending on where it's included, so we don't need to duplicate the example, field tables, etc.